tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -450,6 +451,43 @@ def test_get_metric_extraction_config_with_no_tag_spec(default_project, metric):
         }
 
 
+def test_get_metric_extraction_config_user_misery(default_project, metric):
```

Comment:
As I add the other tests I may have more clarity. Don't look into the body of this test.